components/lead-details.tsx

```diff
@@ -1,55 +0,0 @@
-import React from 'react';
```

Comment:
You have deleted this file, are you sure to do that?

Reply:
I haven’t deleted it. I moved it to a folder named "LeadDetails"